test/rendering/webpack.config.js

```diff
@@ -42,6 +42,11 @@ export default {
     ],
   },
   resolve: {
+    fallback: {
+      fs: false,
+      http: false,
+      https: false,
+    },
```

Comment:
This change is not related to the fix. It simply brings the rendering test webpack config in line with the other webpack configs. This avoids errors when building the rendering tests.